Assets/_BForBoss/_Core/Scripts/EnergySystem/Data/EnergyData.cs

```diff
@@ -17,6 +17,6 @@ float rateOfTransaction
             RateOfTransaction = rateOfTransaction;
         }
 
-        public EnergyData Apply(float value) => new(value, MaxEnergyValue, RateOfTransaction);
+        public EnergyData Apply(float value) => new EnergyData(value, MaxEnergyValue, RateOfTransaction);
```

Comment:
So my Rider kept bugging me about this, as it keeps telling me that this formatting is an error. So just made it explicit to make Rider happy.

Can revert if you want @m4rtint 
![Personal Gripe 2](https://github.com/PerigonGames/BForBoss/assets/23490604/8de15c6f-c0c5-43f5-9e42-5db3876864c2)

Reply:
this is fine.